src/storage/kv/test_engine_builder.rs

```diff
@@ -67,23 +67,30 @@ impl TestEngineBuilder {
     }
 
     /// Build a `RocksEngine`.
-    pub fn build(self) -> Result<RocksEngine> {
+    pub fn build(self, region_id: u64, suffix_id: u64) -> Result<RocksEngine> {
```

Comment:
For builder style, try not to append args in `build`. Instead, you can add this:
```rust
struct TestEngineBuilder {
  ..
  region_id: u64,
  suffix_id: u64,
}

impl TestEngineBuilder {
  pub fn tablet_id(mut self, region_id: u64, suffix_id: u64) -> Self {
    self.region_id = region_id;
    self.suffix_id = suffix_id;
  }
}
```

This way you don't need to change all the legacy tests and make this PR too hard to review.

Reply:
Sorry, I don't get the point here.  Builder can build multiple engines with different region_id and suffxi_id, which shares the same root path, default limiter, api_version, etc.  It seems natural especially in multi-rocksdb scenario. And how can your suggested approach save change to legacy cases ?

Reply:
All the `build(0, 0)` can be reverted.

Reply:
Do you mean give region_id, suffix_id default value 0 for structure TestEngineBuilder?

Reply:
Yes.